docs/platform/api/getting-started-with-the-linode-api-v4.md

```diff
@@ -0,0 +1,358 @@
+            "pvops": true
+        },
+        {
+            "kvm": false,
+            "architecture": "i386",
+            "version": "2.6.18",
+            "xen": true,
+            "label": "2.6.18.8-linode16",
+            "id": "linode/2.6.18.8-linode16",
+            "pvops": false
+        },
+    ]
+        ...
+    "pages": 3
+}
+{{< /file-excerpt >}}
+
+From the response you can see that the results are divided into three pages. View the second page:
+
+    curl https://api.linode.com/v4/linode/kernels?page=2
+
+If you would prefer a smaller number of items per page, you can override the default value with the `page_size` parameter:
+
+    curl https://api.linode.com/v4/linode/kernels?page_size=50
+
+### Filter Results
+
+Version 4 also supports filtering request results. Filters are passed using the `X-Filter` header and use JSON format. You can filter on almost any field that appears in a response object. The following query uses the `deprecated` and `vendor` fields to return all current Debian images:
+
+    curl https://api.linode.com/v4/images/ -H 'X-Filter: { "vendor": "Debian", "deprecated": false}'
+
+  {{< file-excerpt "response.json" json >}}
+{
+    "page": 1,
+    "pages": 1,
+    "data": [
+        {
+            "size": 1024,
+            "type": "manual",
+            "label": "Debian 8",
+            "created_by": "linode",
+            "vendor": "Debian",
+            "is_public": true,
+            "created": "2015-04-27T20:26:41",
+            "deprecated": false,
+            "id": "linode/debian8",
+            "description": ""
+        },
+        {
+            "size": 1100,
+            "type": "manual",
+            "label": "Debian 9",
+            "created_by": "linode",
+            "vendor": "Debian",
+            "is_public": true,
+            "created": "2017-06-16T20:02:29",
+            "deprecated": false,
+            "id": "linode/debian9",
+            "description": null
+        }
+    ],
+    "results": 2
+}
+{{< /file-excerpt >}}
+
+More complex searches are possible through the use of logical operators. Use `or` to return a list of all Debian and Ubuntu images:
+
+    curl https://api.linode.com/v4/images/ -H "{"+or": [{"vendor":"Debian"}, {"vendor":"Ubuntu"}]}"
+
+See the [Linode API documentation](https://developers.linode.com/v4/filtering) for a full list of supported operators.
+
+## Create an API Token and Authenticate Users
+
+Any request that involves accessing personal account information or billable actions will require that an authentication token be sent as a header in the API request. The easiest way to obtain an access token is through the beta [Linode Manager](https://cloud.linode.com).
+
+### Use OAuth for Varying Authentication Levels
+
+If you are building a larger app or need to authenticate users with different levels of access, set up an [OAuth authentication flow](https://developers.linode.com/v4/access) to generate tokens for each user.
```

Comment:
OAuth is primarily useful if you're writing an application for _other people_ to use - if it's just for you, a PAT's fine.  An OAuth application might be a custom interface to Linode's infrastructure for your organization, or it might be a public application that allows users to deploy LInodes with your software pre-configured.